<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-4" name="com.focus.testcases.OpeningStocksTest" tests="26" failures="3" timestamp="23 Jul 2021 15:20:09 GMT" time="264.388" errors="0">
  <testcase name="checkSavingOpeningStockVoucher3With2RowsAndBinItemDoubleClickInIndex" time="19.560" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSaveInCreateViewOfHomePageOfOpeningBalance" time="9.557" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDataOfSavedOpeningStocksVoucher2AndDeleting" time="6.018" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher2AddingRow2WithStockItem" time="6.269" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher1AddingRow2WithBatchItem" time="7.708" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDataOfSavedOpeningStocksVoucher4AndDeleting" time="4.628" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSavingOpeningStockVoucher2WithStockItem" time="5.987" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSavingOpeningStockVoucher1WithBatchItem" time="43.491" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher2" time="2.899" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDataOfSavedOpeningStocksVoucher3AndDeleting" time="3.058" classname="com.focus.testcases.OpeningStocksTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.OpeningStocksTest.checkVerifingDataOfSavedOpeningStocksVoucher3AndDeleting(OpeningStocksTest.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkVerifingDataOfSavedOpeningStocksVoucher3AndDeleting -->
  <testcase name="checkSavingOpeningStockVoucher4WithBinItemByManuallyEnteringQuantity" time="16.456" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher3AddingRow3WithBinItemByAutoAllocation" time="19.888" classname="com.focus.testcases.OpeningStocksTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.OpeningStocksTest.checkEditingAndVerifingDataOfSavedOpeningStocksVoucher3AddingRow3WithBinItemByAutoAllocation(OpeningStocksTest.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkEditingAndVerifingDataOfSavedOpeningStocksVoucher3AddingRow3WithBinItemByAutoAllocation -->
  <testcase name="checkOpeningStockVoucher3Row1WithBinItemWithAutoAllocationOption" time="11.114" classname="com.focus.testcases.OpeningStocksTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.OpeningStocksTest.checkOpeningStockVoucher3Row1WithBinItemWithAutoAllocationOption(OpeningStocksTest.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkOpeningStockVoucher3Row1WithBinItemWithAutoAllocationOption -->
  <testcase name="checkSavingOpeningStockVoucher3WithStockItem" time="4.422" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDetailsOfSuspendedOpeningStocksVouchersInCreatedView" time="4.100" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher4AddingRow2WithRMAItem" time="20.971" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDataOfSavedOpeningStocksVoucher5AndDeleting" time="5.081" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDetailsOfSavedOpeningStocksVouchersInHomepage" time="5.849" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSavingOpeningStockVoucher2WithBatchItem" time="6.788" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSavingOpeningStockVoucher4WithRMAItem" time="16.377" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher4" time="3.483" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSavingOpeningStockVoucher5WithRMAItem" time="16.018" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkVerifingDataOfSavedOpeningStocksVoucher3" time="4.437" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkEditingAndVerifingDataOfSavedOpeningStocksVoucher1" time="3.560" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkSavingAndSuspendingOpeningStockVoucher5WithStockItem" time="9.855" classname="com.focus.testcases.OpeningStocksTest"/>
  <testcase name="checkDeleteOptionInOpeningStocksHomePage" time="6.814" classname="com.focus.testcases.OpeningStocksTest"/>
</testsuite> <!-- com.focus.testcases.OpeningStocksTest -->
